.github/workflows/ci.yml

```diff
@@ -42,5 +41,8 @@ jobs:
       - name: Run linter 👀
         run: yarn lint
 
+      - name: Check if build is up-to-date 🚧
+        run: yarn build && git add --intent-to-add dist && git diff HEAD --quiet -- dist
```

Comment:
`git add --intent-to-add dist` makes sure that we also check files from the dist that have not been committed yet.